src/main/java/sg/ncl/testbed_interface/Team2.java

```diff
@@ -26,7 +30,8 @@
 
     private String organisationType;
     private String status;
-    private String createdDate;
+    private ZonedDateTime applicationDate;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this value-based field transient so it is not included in the serialization of this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3437)